packages/cloud_firestore/cloud_firestore/pubspec.yaml

```diff
@@ -15,6 +15,8 @@ flutter:
         pluginClass: FLTCloudFirestorePlugin
 
 dependencies:
+  cloud_firestore_platform_interface:
+    path: ../cloud_firestore_platform_interface
```

Comment:
Should eventually be pointing to a hosted version (probably by splitting this into two PRs)